openslides/core/static/templates/core/customslide-form.html

```diff
@@ -15,7 +15,7 @@ <h1 ng-if="!customslide.id" translate>New custom slide</h1>
   </div>
   <div class="form-group">
     <label for="textareaDesciption" translate>Text</label>
-    <textarea ng-model="customslide.text" class="form-control" name="textareaText" />
+    <div ckeditor="CKEditorOptions" ng-model="customslide.text"></div>
```

Comment:
we need a new id attribute for the <div> which has the same name as defined in <label for="..">. And please fix the wrong name in "for" attribute.

FYI: "The for attribute of the <label> tag should be equal to the id attribute of the related element to bind them together." (see http://www.w3schools.com/tags/tag_label.asp)